src/main/java/frc/robot/RobotContainer.java

```diff
@@ -57,6 +57,7 @@ public class RobotContainer {
   private OperatorInterface oi = new OperatorInterface() {};
   private RobotConfig config;
   private Drivetrain drivetrain;
+  private Vision vision;
```

Comment:
This instance variable needs to be initialized. Each place in the code where a new Vision subsystem is instantiated, assign the reference to this instance variable. Currently, we will have null pointer exceptions.

Reply:
what should i pass as the parameter for a visionio object when constructing a new vision object is created